addons/mail/static/src/components/messaging_menu/messaging_menu.xml

```diff
@@ -2,23 +2,19 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.MessagingMenu" owl="1">
-        <li class="o_MessagingMenu" t-att-class="{ 'o-is-open': messagingMenu.isOpen, 'o-mobile': env.messaging.device.isMobile }">
-            <!-- TODO FIXME restore mobile stuff here (activeMobileThread) -->
-            <t t-if="!env.messaging.device.isMobile or true">
```

Comment:
I've removed the weird stuff with `isMobile or true` because it was problematic with displaying spinner when `!env.messaging`. We had them just to let us know we should do something for mobile, but it's been too long and the comment and weird boolean expression didn't make much sense IMO.

Reply:
ok